### ISSUE TITLE
New music tracks

### ISSUE THREAD

**Issue by Unit158** [SirCmpwn/TrueCraft#202]

**Comment by SirCmpwn** [comment]
To clarify this issue, @Unit158 has offered to make CC music for TrueCraft's use.

**Comment by SirCmpwn** [comment]
Any update on this?

**Comment by Unit158** [comment]
Yeah, they'll be a while due to a tight schedule with school. If you need one track soon, I can rush one this week.

On 2015-10-26, at 11:34 AM, Drew DeVault <notifications@github.com<mailto:notifications@github.com>> wrote:


Any update on this?

**Comment by SirCmpwn** [comment]
Not in a rush, just curious.
